src/View/ViewableData.php

```diff
@@ -617,9 +629,11 @@ public function hasValue($field, $arguments = [], $cache = true)
      * @param array $arguments
      * @param bool $cache
      * @return string
+     * @deprecated 5.4.0 Will be removed without equivalent functionality to replace it
      */
     public function XML_val($field, $arguments = [], $cache = false)
```

Comment:
This is currently called directly from `SSViewer_Scope` - that will be calling methods on `ViewLayerData` now instead, and the methods on `ViewLayerData` don't rely on this method, so there's no need to keep it.